app/templates/events/event_list.html

```diff
@@ -51,77 +51,123 @@ <h4><strong>Term</strong></h4>
-          </button>
-        </h2>
-        <div id="accordion__body_{{program}}_num_{{ loop.index }}"
-              class="accordion-collapse collapse"
-              aria-labelledby="accordion__header_{{program}}"
-              data-bs-parent="#categoryAccordion">
-          <div class="accordion-body">
-            <table class="table table-striped">
-              <thead>
-                <tr>
-                  <th scope="col">Event Name</th>
-                  <th scope="col">Date</th>
-                  <th scope="col">Time</th>
-                  <th scope="col">Location</th>
-                  <th scope="col">Invitation</th>
-                </tr>
-              </thead>
-              <tbody>
-                  {% for event in events %}
-                    <tr>
-                      <td> <a href="/event/{{event.id}}/edit" class="link-primary">{{event.name}}</a></td>
-                      <td>{{event.startDate.strftime('%m-%d-%Y')}}</td>
-                      <td>{{event.timeStart.strftime('%I:%M %p')}}</td>
-                      <td>{{event.location}}</td>
-                      {% if (user.isCeltsAdmin) or (user.isCeltsStudentStaff) %}
-                        <td><i class="bi bi-envelope-fill"> Email</i></td>
-                        {# FIXME: the Email icon should be clickable and lead to email template #}
-                      {% else %}
-                        {% if (event.startDate < currentTime.date()) or (event.timeStart < currentTime.time() and event.startDate == currentTime.date())%}
-                          <td><button type="button" class="btn btn-primary" disabled>RSVP</button></td>
-                        {% elif event.id in rsvpedEventsID %}
-                          <td><button type="button" class="btn btn-danger" onclick="removeRsvpForEvent({{program.id}}, {{event.id}})">Remove RSVP</button></td>
+{% macro createTable(events) %}
+{% if events.exists() %}
+  <table class="table table-striped">
+    <thead>
+      <tr>
+        <th scope="col">Event Name</th>
+        <th scope="col">Date</th>
+        <th scope="col">Time</th>
+        <th scope="col">Location</th>
+        <th scope="col">Invitation</th>
+      </tr>
+    </thead>
+    <tbody>
+
+        {% for event in events %}
+          <tr>
+            <td> <a href="/{{event.program_id}}/{{event.id}}/edit_event" class="link-primary">{{event.name}}</a></td>
+            <td>{{event.startDate.strftime('%m-%d-%Y')}}</td>
+            <td>{{event.timeStart.strftime('%I:%M %p')}}</td>
+            <td>{{event.location}}</td>
+            {% if (user.isCeltsAdmin) or (user.isCeltsStudentStaff) %}
+              <td><i class="bi bi-envelope-fill"> Email</i></td>
+              {# FIXME: the Email icon should be clickable and lead to email template #}
+            {% else %}
+              {% if (event.event.startDate < currentTime.date()) or (event.event.timeStart < currentTime.time() and event.event.startDate == currentTime.date())%}
```

Comment:
we'll use the `event.isPast` shortcut if it's available by the time this PR is ready. we'll see.